src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -309,23 +312,62 @@ private void Initialize()
 		}
 
 		IntPtr window;
+		uint depth = 32;
```

Comment:
This variable is unused. If you still want to use it instead of keeping the hardcoded `32` below, I would make it a const at the top of the class.

Reply:
@ramezgerges Thank you, but this variable will be bring to X11SoftwareRenderer.